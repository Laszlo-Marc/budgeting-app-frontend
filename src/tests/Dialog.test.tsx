import {render} from '@testing-library/react';
import ExpenseDialog from '../components/expenseComponents/Dialog';

describe('render ExpenseDialog component corectly', () => {
    it('should render ExpenseDialog component correctly', () => {
        const {container} = render(<ExpenseDialog />);
        expect(container.firstChild).toMatchSnapshot();
    });
});
