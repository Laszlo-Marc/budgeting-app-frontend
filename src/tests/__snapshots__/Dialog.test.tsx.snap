// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`render ExpenseDialog component corectly > should render ExpenseDialog component correctly 1`] = `null`;
